ckan/views/group.py

```diff
@@ -958,7 +964,13 @@ def post(self, group_type, is_organization, id=None):
             group = _action(u'group_update')(context, data_dict)
             if id != group['name']:
                 _force_reindex(group)
-
+            if group_type == u'organization':
+                h.flash_success(_(u'Organization updated.'))
+            elif group_type == u'group':
+                h.flash_success(_(u'Group updated.'))
+            else:
+                h.flash_success(
+                    _(u'%s updated.') % _(group_type.capitalize()))
```

Comment:
We're not using other group types and this isn't translatable the way it's written. Remove the else?

Reply:
Okay, just removed the group type condition and doing `is_organization` for these messages.